src/elements/Shape.tsx

```diff
@@ -65,7 +64,7 @@ export interface SVGMatrix extends Matrix {
   skewY(angle: number): SVGMatrix;
 }
 
-export function multiply_matrices(l: Matrix, r: Matrix): Matrix {
+export function multiplyMatrices(l: Matrix, r: Matrix): Matrix {
```

Comment:
Why does this name change? Is it a private or public function?

Reply:
It is exported but not used anywhere in the codebase and not exported outside of repo so it shouldn't be a problem.